ckanext/versioning/logic/action.py

```diff
@@ -372,8 +370,8 @@ def _get_package_in_revision(context, data_dict, revision_id):
         backend = get_metastore_backend()
         dataset_name = _get_dataset_name(data_dict.get('id'))
         pkg_info = backend.fetch(dataset_name, revision_id)
-        dp = datapackage.DataPackage(pkg_info.package)
-        result.update(converter.datapackage_to_dataset(dp))
+        dataset = frictionless_to_dataset(pkg_info.package)
```

Comment:
I'd add a short comment for future developers explaining the logic here of what we are doing e.g.

```
# first we convert metastore dataset in Frictionless format to CKAN dataset
# then we merge that with the internal CKAN dataset (package) from CKAN internal DB
# why do we do the merge?
# b/c the MetaStore dataset is "clean" and does not have things like `datastore_active` that aren't "revisioned"
# over time we hope to refactor this more cleanly with presentation info like `datastore_active` moving out of logic layer more info presentation layer